WcaOnRails/config/locales/fr.yml

```diff
@@ -1267,9 +1267,9 @@ fr:
       website: Site internet
       #original_hash: 680567f
       organizer_plural:
-        other: Organisateurs
-        one: Organisateur
         zero: Organisateur
+        one: Organisateur
```

Comment:
Actually it's related with saving to mongodb, I remember I managed to prevent from reordering of the whole objects, but looks like things are still messy for the pluralization keys. I'll try to find a solution.